packages/beeq/src/components/tag/scss/bq-tag.variables.scss

```diff
@@ -20,19 +20,19 @@
    * @prop --bq-tag--medium-padding-x: Tag medium padding horizontal
    * @prop --bq-tag--medium-padding-y: Tag medium padding vertical
    */
-  --bq-tag--background-color: theme('colors.ui.secondary');
+  --bq-tag--background-color: theme(colors.ui.secondary);
 
-  --bq-tag--border-color: theme('colors.transparent');
-  --bq-tag--border-radius: theme('borderRadius.s');
+  --bq-tag--border-color: theme(colors.transparent);
+  --bq-tag--border-radius: theme(borderRadius.s);
   --bq-tag--border-style: solid;
   --bq-tag--border-width: theme('borderWidth.s');
 
-  --bq-tag--small-border-radius: theme('borderRadius.xs');
-  --bq-tag--small-gap: theme('spacing.xs2');
+  --bq-tag--small-border-radius: theme(borderRadius.xs);
+  --bq-tag--small-gap: theme(spacing.xs2);
   --bq-tag--small-padding-x: calc(#{theme('spacing.xs')} - var(--bq-tag--border-width));
   --bq-tag--small-padding-y: calc(#{theme('spacing.xs3')} - var(--bq-tag--border-width));
```

Comment:
Considering that we cannot use theme() without quotes here, it might be better to use the CSS variable directly instead.

```scss
  --bq-tag--small-padding-x: calc(var(--bq-spacing.xs) - var(--bq-tag--border-width));
  --bq-tag--small-padding-y: calc(var(--bq-spacing.xs3) - var(--bq-tag--border-width));
```

You might need to make a small change  in the `packages/beeq/.stylelintrc.json` file, line 10, and allow numbers as well:

```diff
- "custom-property-pattern": "^bq(?:(-{1,2})[a-zA-Z]+)+$",
+ "custom-property-pattern": "^bq(?:(-{1,2})[a-zA-Z0-9]+)+$",
```